src/EditorFeatures/Test2/FindReferences/FindReferencesTests.AnonymousTypeSymbols.vb

```diff
@@ -165,5 +166,57 @@ End Class
 </Workspace>
             Await TestAPIAndFeature(input, kind, host)
         End Function
+#End Region
+
+#Region "C#"
+        <WorkItem("https://github.com/dotnet/roslyn/issues/20115")>
+        <WpfTheory, CombinatorialData>
+        Public Async Function TestAnonymousTypesCSharp_SimpleExpression(kind As TestKind, host As TestHost) As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+            <![CDATA[
+class C
+{
+    void M(string {|Definition:a|})
+    {
+        int length = [|a|].Length;
+        var r = new { [|a|], Length = 1 };
+        r = new { $$[|a|], [|a|].Length };
```

Comment:
add an example where there is a named-arg for 'a'.  like `new { a = "", ...}`

Reply:
add test where the $$ is on the defintion.  add test where $$ is on the `a` in `new { a = "", ...`

Reply:
This is already done, marking as resolved